src/NuGet.Indexing/Extraction/MetadataConstants.cs

```diff
@@ -17,6 +17,7 @@ public static class LuceneMetadata
             public const string NormalizedVersionPropertyName = "Version";
             public const string FullVersionPropertyName = "FullVersion";
             public const string VerbatimVersionPropertyName = "OriginalVersion";
+            public const string CaseInsensitiveNormalizedVersionPropertyName = "CaseInsensitiveVersion";
```

Comment:
Could `CaseInsensitiveNormalizedVersion` be a better string value here?

Reply:
Yes, I like that better.